modules/swagger-codegen/src/main/java/io/swagger/codegen/CliOption.java

```diff
@@ -20,4 +33,49 @@ public String getDescription() {
     public void setDescription(String description) {
         this.description = description;
     }
+
+    public String getType() {
+        return type;
+    }
```

Comment:
Please add `@ApiModelPropterty` annotation to describe what `type` means.  I know you mean the JSON Schema data type, but end-users may not.